src/environmentd/src/http/sql.rs

```diff
@@ -99,6 +107,28 @@ async fn run_ws(state: &WsState, mut ws: WebSocket) {
         ))
         .await;
 
+    // Send any notices that might have been generated on startup.
+    let notices = client
```

Comment:
Can this become a function since it's called multiple places and does some specific string munging?

Reply:
Good call! Updated this